scanpy/plotting/_tools/scatterplots.py

```diff
@@ -106,6 +106,17 @@ def embedding(
     Returns
     -------
     If `show==False` a :class:`~matplotlib.axes.Axes` or a list of it.
+
+    Examples
+    --------
+
+    .. plot::
+        :context: close-figs
+
+        >>> import scanpy as sc
```

Comment:
Do you want to also remove the ">>>" prefix here as in other examples?

Reply:
I was actually planning on removing this one, just because I think it's not a very useful example. I have some idea's about how we should handle example for the embedding plots (since they share so much, and are fairly complex) but I'll tag you in another issue about that.